modules/visual_script/doc_classes/VisualScriptBuiltinFunc.xml

```diff
@@ -204,7 +204,10 @@
 		<constant name="COLORN" value="62" enum="BuiltinFunc">
 			Return the [Color] with the given name and alpha ranging from 0 to 1. Note: names are defined in color_names.inc.
 		</constant>
-		<constant name="FUNC_MAX" value="63" enum="BuiltinFunc">
+		<constant name="MATH_SMOOTHSTEP" value="63" enum="BuiltinFunc">
+			Return a number smoothly interpolated between the first two inputs, based on the third input. Using Hermite interpolation formula : float t; t = clamp((weight - from) / (to - from), 0.0, 1.0); return t * t * (3.0 - 2.0 * t);
```

Comment:
- No space before `:`
- Simpler with `float t = clamp(...` directly
- For those not familiar with C++, GDScript syntax might be clearer
- Use `[code]` formatting, or better, a `[codeblock]`:
```
[codeblock]
var t = clamp((weight - from) / (to - from), 0.0, 1.0)
return t * t * (3.0 - 2.0 * t)
[/codeblock]
```
The `@GDScript` version should likely be synced with this one. Or add the details to `@GDScript`, and link to `[method @GDScript.smoothstep]` for details.

Reply:
Fixed !

Reply:
I changed it to the GDScript version but with example.. Seems ok ?

Reply:
Oh.. wait.. I think I did it wrong

Reply:
Hmm.. ok I changed the docs. They are more simular to each other now,  please review again